src/services/netMgr.js

```diff
@@ -203,17 +222,31 @@ NetMgr.setLocalStorageFromToken = function(token) {
 
 // Add interceptor to detect an expired access_token and refresh;
 NetMgr.axiosInstance.interceptors.response.use(
-    function (response) {
+    (response) => {
```

Comment:
We are compiling down to ES5 anyway ++ Charlie finds the syntax easier to read. Probably to be consistent we use ES5 here are ES6 in tests? Until we get a standard to enforce - no sense changing arbitrarily.